### PR TITLE
changed lets to var

### PR DESCRIPTION
**Pull request by JamieZafar**

### CODE REVIEW

File: src/client/lobby/user-list.spec.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { shallow } from 'enzyme';
 
 import UserList from './user-list';
 
-let mockList = [{id: 0, name: 'Jamie'}, {id: 1, name: 'Raul'}];
+var mockList = [{id: 0, name: 'Jamie'}, {id: 1, name: 'Raul'}];
```
Comment (lclarksl) [inline src/client/lobby/user-list.spec.js]: Why has this changed?

Comment (JamieZafar) [inline src/client/lobby/user-list.spec.js]: heroku might have problem with lets

Comment (lclarksl) [inline src/client/lobby/user-list.spec.js]: Only on the backend though?

Comment (JamieZafar) [inline src/client/lobby/user-list.spec.js]: we only used let on the backend

Comment (JamieZafar) [inline src/client/lobby/user-list.spec.js]: maybe, dunno, just doin what kenny tells us to

**Pull request closed without merging**